packages/client/src/components/Input.re

```diff
@@ -0,0 +1,80 @@
+module Text = {
+  [@react.component]
+  let make =
+      (
+        ~inputRef=React.useRef(Js.Nullable.null)->ReactDOMRe.Ref.domRef,
+        ~onChange=?,
+        ~placeholder,
+        ~id,
+        ~label,
+        ~type_="text",
+      ) => {
+    <div className="mb-4">
+      <label htmlFor=id className="block text-gray-700 text-sm font-bold mb-2">
+        label->React.string
```

Comment:
We could lock this to be of type `I18n.Translations.t` if we'd like?

```suggestion
        {I18n.Translations.(toString(label))->React.string}
```

Reply:
Good idea!